cloud/docker-image/pom.xml

```diff
@@ -344,25 +344,25 @@
         </dependency>
         <dependency>
           <groupId>${project.groupId}</groupId>
-          <artifactId>validator-avro</artifactId>
+          <artifactId>type-avro</artifactId>
```

Comment:
We had agreed on `types-`, not `type-`.
Please adjust the module names and module-info accordingly.